tests/unifiedPlatform/pds2/pds_basic_test.go

```diff
@@ -116,6 +116,35 @@ var _ = BeforeSuite(func() {
 		log.FailOnError(err, "Unable to Install pds on destination target cluster")
 	})
 
+	Step("Create Service Configuration, Resource and Storage Templates", func() {
+		var err error
+		WorkflowPDSTemplate.Platform = WorkflowPlatform
+		DsNameAndAppTempId, StTemplateId, ResourceTemplateId, err = WorkflowPDSTemplate.CreatePdsCustomTemplatesAndFetchIds(NewPdsParams)
+		log.FailOnError(err, "Unable to create Custom Templates for PDS")
+
+		for _, AppTemplateId := range DsNameAndAppTempId {
+			TemplateIds = append(TemplateIds, AppTemplateId)
+		}
+		TemplateIds = append(TemplateIds, ResourceTemplateId, StTemplateId)
+
+		for _, tempId := range TemplateIds {
+			log.Debugf("TemplateID: [%s]", tempId)
+		}
+	})
+
+	Step("Associate templates to the Project", func() {
```

Comment:
Could you please move this to the current associate method instead of associating the resources twice?

Reply:
As discussed, I am resolving this